mod/cloud_data_storage/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "elgg/elgg-cloud-storage",
```

Comment:
elgg/cloud-data-storage? None of the core elgg repos repeat elgg on the right hand side of the name.